app/src/main/java/com/github/polypoly/app/map/LocalizationRepository.kt

```diff
@@ -0,0 +1,94 @@
+package com.github.polypoly.app.map
+
+import com.github.polypoly.app.game.Localization
+import com.github.polypoly.app.game.LocalizationLevel
+import com.github.polypoly.app.game.Zone
+import org.osmdroid.util.GeoPoint
+
+/**
+ * Repository for providing the zones and localizations on the map.
+ */
+object LocalizationRepository {
+
+    /**
+     * List of localizations in the EPFL campus area.
+     */
+    private val localizations = listOf(
+        Localization(
+            name = "BC",
+            basePrice = 200,
+            baseTaxPrice = 20,
+            baseMortgagePrice = 50,
+            currentLevel = LocalizationLevel.STUDY_ROOM,
```

Comment:
All localizations are initialized to EMPTY at the beginning

Reply:
As stated above this is all copilot stuff I did not even look at twice, so it will be leaving as discussed in the above comment anyways.